src/tuplet.ts

```diff
@@ -273,19 +273,32 @@ export class Tuplet extends Element {
       // y_pos = first_note.getStemExtents().topY - 10;
 
       for (let i = 0; i < this.notes.length; ++i) {
```

Comment:
Can you add a comment around what this loop is doing?

Reply:
done